ci/tests/Jenkinsfile.e2e-prs

```diff
@@ -84,6 +84,7 @@ pipeline {
       steps { script {
         dir('test/appium') {
           sh "python3 -m venv ${VIRTUAL_ENV}"
+          sh 'pip3 install kiwisolver==1.3.1 --use-pep517'
```

Comment:
Why are we adding this here and not in `requirements.txt`?

Reply:
That was a temp fix for linux-03, where the builds failed because of this package.
I'll remove it now, thanks for prompt!